ride-share-app.rb

```diff
@@ -0,0 +1,253 @@
+ride_share_trips = [
+    {
+        'DR001' =>  [
+            {
+                date: '3rd_Feb_2016',
+                cost: 10.00,
+                rider: 'RD0003',
+                rating: 3.0
+            },
+            {
+                date: '3rd_Feb_2016',
+                cost: 30.00,
+                rider: 'RD0015',
+                rating: 4.0
+            },
+            {
+                date: '5th_Feb_2016',
+                cost: 45.00,
+                rider: 'RD0003',
+                rating: 2.0
+            }     
+        ],
+        'DR002' => [
+            {
+                date: '3rd_Feb_2016',
+                cost: 25.00,
+                rider: 'RD0073',
+                rating: 5.0
+            },
+            {
+                date: '4th_Feb_2016',
+                cost: 15.00,
+                rider: 'RD0013',
+                rating: 1.0
+            },
+            {
+                date: '5th_Feb_2016',
+                cost: 35.00,
+                rider: 'RD0066',
+                rating: 3.0
+            },
+        ],
+        'DR003' => [
+            { 
+                date: '4th_Feb_2016',
+                cost: 5.00,
+                rider: 'RD0066',
+                rating: 5.0
+            },
+            { 
+                date: '5th_Feb_2016',
+                cost: 50.00,
+                rider: 'RD0003',
+                rating: 2.0
+            } 
+        ],
+        'DR004' => [
+            { 
+                date: '3rd_Feb_2016',
+                cost: 5.00,
+                rider: 'RD0022',
+                rating: 5.0      
+            },
+            {
+                date: '4th_Feb_2016',
+                cost: 10.00,
+                rider: 'RD0022',
+                rating: 4.0
+            },
+            {
+                date: '5th_Feb_2016',
+                cost: 20.00,
+                rider: 'RD0073',
+                rating: 5.0
+            }
+        ]
+    
+    }
+] 
+
+#prints to terminal the number of rides each driver has given
+puts "---------------------------------------------------"
+puts "How many trips did each driver make?"
+puts "\n"
+
+def driver_trips(ride_share_rides)
```

Comment:
I like the methods here, but I suggest putting the method bodies/definitions at the top of the file and the main program code below, so the method bodies don't break up the flow of the program.  Let me know if this is unclear.